### ISSUE TITLE
Updating full frame buffer causes render glitch.

### ISSUE THREAD

**Issue by XV-02** [pop-os/plymouth#1]
With this plymouth implementation as of commit 23915e9, updating the full frame buffer during the decrypt screen can cause the following graphical glitch:

https://user-images.githubusercontent.com/98765732/156240722-9c465849-8d73-49f4-ab3b-e3a5126ac684.mp4

The video shows an implementation with the cursor set to "breath" causing constant updating. 

Even with a static cursor, the rendering of pips during password input caused the same issue, though each individual input - redrawing the screen only once - left the stray, static like artefacts in a constant position.

This was found using a GT 710 card with nouveau drivers. The issue was not present with Nvidia 470 drivers.

**Comment by leviport** [comment]
This is a bug, so we'll leave it open, but I want to be clear that it is extremely low priority to work on. We made sure 470 was available for install for users with GT700 series GPUs, so most users likely won't be using Nouveau.
